tests/integration/test.sync.js

```diff
@@ -332,11 +332,11 @@ adapters.forEach(function (adapters) {
       replications.cancel();
     });
 
-    it.skip('Test sync cancel called twice', function (done) {
+    it('Test sync cancel called twice', function (done) {
       var db = new PouchDB(dbs.name);
       var remote = new PouchDB(dbs.remote);
       var replications = db.sync(remote).on('complete', function () {
-        setTimeout(done); // let cancel() get called twice before finishing
+        done();
```

Comment:
Can we take this out? this worries me for 2 reasons

`.cancel()` caused intermittent crashes in newer nodes so these tests had to be skipped to get back to a stable tree, the other is that comment indicates that the setTimeout shouldnt be removed (also indicates that this isnt the most stable test), the `complete` is going to be triggered by the first call to `.cancel`, if we then call test immediately then the code is almost certainly going to still be running while the test suite moves onto the next test.

We can still do this, but I would prefer it go into a new PR and we do a large node test run to ensure we dont bring back the issues that caused us to skip the test, both those previous issues causes non reliable test failures that are super hard to track down